app/models/comment.rb

```diff
@@ -4,4 +4,9 @@ class Comment < ActiveRecord::Base
 
   validates_presence_of :user
   validates_presence_of :parent
+
+  after_create :send_new_comment_notification
+  def send_new_comment_notification
+    CommentMailer.send_notification_email(self.id).deliver!
```

Comment:
Since we are now using Sidekiq, use `deliver_later` instead. However, we may need to upgrade to Rails 4.2 for this to work.